spec/system/invitations/user_view_invitation_details_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+describe 'Usuário vê detalhes de um convites' do
+  include ActiveSupport::Testing::TimeHelpers
+
+  it 'a partir da página inicial' do
+    owner = create :user
+    project = create :project, user: owner, title: 'Projeto Top', category: 'Video'
+    second_project = create :project, user: owner, title: 'Projeto Master', category: 'Aplicação WEB',
+                                      description: 'Um projeto sobre masterizar tudo que é possível'
+    create :project, user: owner, title: 'Projeto'
+    user = create :user, cpf: '000.000.001-91'
+    profile = PortfoliorrrProfile.new(id: 92, name: 'Pedro', job_categories: [JobCategory.new(name: 'Designer')])
+    create :invitation, profile_id: profile.id, project:, profile_email: user.email,
+                        message: 'Gostaria de te convidar', expiration_days: 8
+    invitation = create :invitation, profile_id: profile.id, project: second_project, profile_email: user.email,
+                                     message: 'Por favor aceite', expiration_days: 5
+
+    login_as user
+    visit root_path
+    click_on 'Convites'
+    click_on 'Projeto Master'
+
+    expect(current_path).to eq invitation_path invitation
```

Comment:
Uma prática boa de adotar nesses casos é usar `expect(page).to have_current_path` que chama o comportamento assíncrono do Cuprite. A chamada do `current_path eq` é até considerada uma ofensa pelo rubocop_capybara pois pode causar inconsistências no teste.

Reply:
O rubocop não reclamou aqui, mas vou trocar, tentar manter o padrão do projeto.